website/docs/install-deploy/deploying-distributed-cluster.md

```diff
@@ -70,62 +74,85 @@ cd fluss-$FLUSS_VERSION$/
 
 ### Configuring Fluss
 
-After having extracted the archived files, you need to configure Fluss for the cluster by editing `conf/server.yaml`
+After having extracted the archived files, you need to configure Fluss for a distributed deployment.
+We will use the _default config file_ (`conf/server.yaml`) to configure Fluss.
+Adapt the `server.yaml` on each node as follows.
+
+**Node0**
 
-For **Node1**, the config is as follows:
-```yaml
-coordinator.host: 192.168.10.1
-coordinator.port: 9123
-zookeeper.address: 192.168.10.99:2181
+```yaml title="server.yaml"
+# coordinator server
+bind.listeners: FLUSS://192.168.10.100:9123
+
+zookeeper.address: 192.168.10.199:2181
 zookeeper.path.root: /fluss
 
-tablet-server.host: 192.168.10.1
-tablet-server.id: 1
+remote.data.dir: /tmp/fluss-remote-data
 ```
 
-For **Node2**, the config is as follows:
-```yaml
-zookeeper.address: 192.168.10.99:2181
+**Node1**
+
+```yaml title="server.yaml"
+# tablet server
+bind.listeners: FLUSS://192.168.10.101:9124 # alternatively, setting the port to 0 assigns a random port
```

Comment:
We can use `9123` port to align with coordinator port as we are using different nodes for each server.